Java/src/com/couchbase/litecore/C4Database.java

```diff
@@ -170,13 +178,20 @@ public C4Document getBySequence(long sequence) throws LiteCoreException {
 
     // - Purging and Expiration
 
-    public void setExpiration(String docID, long timestamp)
+    public boolean setExpiration(String docID, Long timestamp)
```

Comment:
The signature should be `public void setExpiration(String docID, long expiration)`.